internals/cli/wait.go

```diff
@@ -31,9 +31,9 @@ var (
 )
 
 type waitMixin struct {
-	clientMixin
 	NoWait    bool `long:"no-wait"`
 	skipAbort bool
+	client    *client.Client
```

Comment:
I think we have a slight inconsistency left here, which is not critical, but I will point it out in anycase.

1. We require the remaining ```MiXin``` helpers to include a client, which feels a bit odd. These ```MiXin``` helpers are used to include a common set of functionality often used.
2. All of a sudden, if you wish to have some common args, you need to leave out the command client definition, for some of the ```MiXins``` in the command struct, and even not all of them.
3. By specifying it inside the ```MiXin```, we make these helpers ugly to export in the future, because unlike the external command that may use a custom client, they have to then be setup with a Pebble client (they are private today - should they be Public for use by external commands - maybe later on demand if we really need them)?

So here are my thoughts:

- Leave them private until we need them
- Remove the client definition inside the ```MiXin``` struct, and rather supply it to the member.
- This also allows multiple helper ```MiXins``` to be cleaner to support in the same command.

e.g:

```go
func (wmx waitMixin) wait(client client.Client, id string) (*client.Change, error) {
```